contracts/hip-583/ContractTransferTx.sol

```diff
@@ -8,8 +8,6 @@ contract ContractTransferTx {
     function transferTo(address payable _to, uint256 _amount) public {
         require(address(this).balance > _amount, "Insufficient contract balance");
 
-        _to.call{value : _amount}("");
```

Comment:
Q: Why was this removed?

Reply:
How can this be true? 
Without this line this method only does a require and then an event emission. 
If it's not being used then where is the transfer done?